src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
```

Comment:
```suggestion
import static core.basesyntax.db.Storage;
```
Below use the Storage.people entry so it is more readable